src/vivarium_conic_lsff/components/observers.py

```diff
@@ -132,4 +132,98 @@ def get_transition_count(pop, config, disease, transition, event_time, age_bins)
+                'by_year': True,
+                'by_sex': True,
+            }
+        }
+    }
+
+    def __init__(self):
+        self.disease = project_globals.NTD_MODEL_NAME
+        self.configuration_defaults = {
+            'metrics': {f'{self.disease}_observer': LiveBirthWithNTDObserver.configuration_defaults['metrics'][
+                'disease_observer']}
+        }
+
+    @property
+    def name(self):
+        return f'live_birth_with_ntd_observer'
+
+    def setup(self, builder):
+        self.config = builder.configuration['metrics'][f'{self.disease}_observer'].to_dict()
+        self.live_birth_counts = Counter()
+        self.with_disease = Counter()
+        tmp = builder.configuration['time']['start']
+        self._reference_time = pd.Timestamp(tmp['year'], tmp['month'], tmp['day'])
+
+        columns_required = ['alive', f'{self.disease}', 'entrance_time']
+        if self.config['by_sex']:
+            columns_required.append('sex')
+        self.population_view = builder.population.get_view(columns_required)
+
+        builder.value.register_value_modifier('metrics', self.metrics)
+        builder.event.register_listener('collect_metrics', self.on_collect_metrics)
+
+    def on_collect_metrics(self, event):
+        pop = self.population_view.get(event.index)
+        self.live_birth_counts.update(
+            get_live_births(pop, self.config, self._reference_time, event.time.year))
+        self.with_disease.update(
+            get_with_ntd_births(pop, self.config, self._reference_time, event.time.year))
+        self._reference_time = pd.Timestamp(event.time.year, event.time.month, event.time.day)
+
+    def metrics(self, index, metrics):
+        metrics.update(self.live_birth_counts)
+        metrics.update(self.with_disease)
+        return metrics
+
+    def __repr__(self):
+        return f"DiseaseObserver({self.disease})"
+
+
+def get_live_births(pop, config, reference_time, year):
+    retval = {}
+    filter = QueryString(f'alive == "alive" and entrance_time >= "{reference_time}"')
+    base_key = get_output_template(False, **config).substitute(
+        measure=f'live_births', year=year if config['by_year'] else None)
+    if config['by_sex']:
+        for i in ["Male", "Female"]:
+            filt_sex = filter + f'sex == "{i}"'
+            df = pop.query(filt_sex)
+            key = f'{base_key.substitute(sex=i)}'
+            retval[key] = len(df)
+    else:
+        df = pop.query(filter)
+        retval[f'{base_key}'] = len(df)
+    return retval
+
+
+def get_with_ntd_births(pop, config, reference_time, year):
+    retval = {}
+    filter = QueryString(f'alive == "alive"'
+            f' and {project_globals.NTD_MODEL_NAME} == "{project_globals.NTD_MODEL_NAME}"'
+            f' and entrance_time >= "{reference_time}"')
+    base_key = get_output_template(False, **config).substitute(
+        measure=f'born_with_ntd', year=year if config['by_year'] else None)
+    if config['by_sex']:
```

Comment:
Same as above.  This duplicates what get_group_counts does.

Reply:
OK, I looked at get_group_counts() and it wasn't clear that I could use it without the age bins. I'll switch things to use it.